db/src/main/kotlin/edu/artic/db/models/ArticEvent.kt

```diff
@@ -4,21 +4,31 @@ import android.arch.persistence.room.Entity
 import android.arch.persistence.room.PrimaryKey
 import android.os.Parcelable
 import com.squareup.moshi.Json
+import edu.artic.base.utils.getLocalDateTime
 import kotlinx.android.parcel.Parcelize
-import org.threeten.bp.LocalDateTime
+import org.threeten.bp.ZonedDateTime
 
 @Entity
 @Parcelize
 data class ArticEvent(
         @Json(name = "short_description") val short_description: String?,
         @Json(name = "image") val image: String?,
-        @Json(name = "end_at") val end_at: LocalDateTime,
+        @Json(name = "end_at") val end_at: ZonedDateTime,
         @Json(name = "button_url") val button_url: String?,
         @Json(name = "description") val description: String?,
         @Json(name = "location") val location: String?,
         @Json(name = "id") @PrimaryKey val id: Int,
         @Json(name = "button_text") val button_text: String?,
         @Json(name = "title") val title: String,
-        @Json(name = "start_at") val start_at: LocalDateTime
+        @Json(name = "start_at") val start_at: ZonedDateTime
 
-) : Parcelable
+) : Parcelable {
+    fun getStartTime(): ZonedDateTime {
```

Comment:
property syntax :) please lets move away from getter/setter thinking and more property thinking.

Reply:
Updated. :)